doc/dependencies.md

```diff
@@ -0,0 +1,24 @@
+# Dependencies
+
+## Run Time Dependencies
+
+| Dependency         | Purpose                      | License |
+|--------------------|------------------------------|---------|
+| [Python 3][python] | Python version 3.8 and above | PSF     |
+| t.b.d.             | ...                          | ...     |
+
+
+## Test Dependencies
+
+| Dependency       | Purpose           | License |
+|------------------|-------------------|---------|
+| [Pytest][pytest] | Testing framework | MIT     |
+| ...              | ...               | ...     |
+
+
+
+[python]: https://docs.python.org
+t.b.d.
+
+[pytest]: https://docs.pytest.org/en/stable/
```

Comment:
Rather list the non python dependencies or what is needed to bootstrap/run the project, e.g.:
* Python >= 3.8
* Poetry >= X.Y
* ...

The actually list is anyhow part of the project `pyproject.toml` if will still want to have it explicit in the documentation I would consider generating this.

Reply:
I removed the file for now. As discussed offline, should created automatically by a script.